src/resources/utils/arrayUtils.tsx

```diff
@@ -8,3 +13,39 @@ export function arraysEqual<T>(array1?: T[], array2?: T[]): boolean {
   }
   return true;
 }
+
+/**
+ * Paginates an array, returning the array elements pertaining to the given page.
+ * @param array The array to be paginated.
+ * @param page The page number of the returned elements. (Indexation starts at 1)
+ * @param numPerPage Number of elements on each page.
+ * @returns The elements present on the given page.
+ * @throws {RangeError} Thrown when either numPerPage or page is not valid (e.g., asking for a page that does not exist).
+ */
+export function getArrayPage<T>(array: T[], page: number, numPerPage: number): T[] {
+  if (numPerPage <= 0) {
+    throw new RangeError('numPerPage must be greater than 0');
+  }
+
+  if (page <= 0 || page > Math.ceil(array.length / numPerPage)) {
```

Comment:
Can we reuse `getTotalNumOfPages` here? 😄

```suggestion
  if (page <= 0 || page > getTotalNumOfPages(array, numPerPage)) {
```

Reply:
Let's do it! 😄